Yoast/yoast-components
# Change initial state to Open

Remove conditional isOpen state variables
Add additional test for collapsed state snapshot
Update snapshots

## Summary

This PR can be summarized in the following changelog entry:

* Changes the default state for Analysis headers to Collapsed instead of conditionally based upon the most important issues.

## Relevant technical choices:

* Removed the value from the components when they match the default.

## Test instructions

This PR can be tested by following these steps:

* Open the Analysis and see the headings being expanded.
* The headings should still be collapsible.

Fixes #430
